app/apps/server/bridges/rooms.js

```diff
@@ -121,6 +121,11 @@ export class AppRoomBridge {
 		}
 	}
 
+	async delete(roomId, appId) {
+		this.orch.debugLog(`The App ${ appId } is deleting a room.`);
+		Rooms.removeById(roomId);
```

Comment:
This should have an `await` as it access the db
```suggestion
		await Rooms.removeById(roomId);
```

Reply:
Nevermind. The module does not use awaits at all, so let's stay consistent. This will be changed in the refactor, though